### PR TITLE
[#140810997] added and used new Header component

### PR DESCRIPTION
**Pull request by merlinex**
- created a new Header component
- used Header component in main and photo details pages

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -29,7 +30,7 @@ class App extends Component {
   render() {
     return (
       <main>
-        <h3>Photo Map App</h3>
+        <Header/>
```
Comment (melaniebrgr) [inline src/components/App/App.js]: The `<Header />` should probably be outside the `<main>` element, semantically. You'll need a wrapper `<div>` as well.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export default function Header({
+}) {
+  return (
+    <h3>Photo Map App</h3>
```
Comment (melaniebrgr) [inline src/components/Header/Header.js]: Let's change this to an `<h1>` too – I think this is better for accessibility.

**Review by melaniebrgr — CHANGES_REQUESTED** [review]
Nice work, Merline!

**Review by melaniebrgr — APPROVED** [review]

**Comment by merlinex** [comment]
thanks @melaniebrgr; rebased and made the recommended changes

**Pull request merged**